app/lib/contexts.json

```diff
@@ -0,0 +1,7 @@
+{
+  "stomachAche": {
+    "context": "stomach-ache",
```

Comment:
Not sure if `context` is the best key name here? I was thinking `key` might be a better option but then thought that was too generic so left it as is.